patroni/postgresql.py

```diff
@@ -660,7 +661,7 @@ def create_replica(self, clone_member):
                     break
             # if the method is basebackup, then use the built-in
             if replica_method == "basebackup":
-                ret = self.basebackup(connstring, env)
+                ret = self.basebackup(connstring, env, copy_list_dict(self.config.get(replica_method) or {}))
```

Comment:
I don't really understand why do you want to make a copy of `config[replica_method]`, `basebackup` doesn't change anything in it...
In case if you really want to protect it from changes, it makes sense to use `copy.deepcopy` instead of reinventing the wheel.

Reply:
Seems to be a stupid copy/paste thing. Thanks.